feathr_project/feathr/api/requirements.txt

```diff
@@ -0,0 +1,8 @@
+azure-core
+azure-purview-catalog==1.0.0b2
+fastapi
+opencensus-ext-azure
+pyapacheatlas
+pydantic
+uvicorn
+git+https://github.com/linkedin/feathr.git@main#subdirectory=feathr_project
```

Comment:
could you list the licenses in the PR description and mention if they are compatible with ours. 

What's the last-line for?